src/fragments/ui/auth/flutter/authenticator.mdx

```diff
@@ -0,0 +1,335 @@
+## Basic Integration
+
+The Amplify-Flutter Authenticator Widget is a Material-compatible sign-up and sign-in experience which reacts to your Amplify configuration. It automatically detects the authentication state and updates the UI. If the user is signed in, the underlying component (typically your app's home widget) is displayed; otherwise, signin/signup controls are displayed.
+
+To begin, follow the steps for configuring the Amplify Auth category from the [getting started walkthrough](/lib/auth/getting-started). Be sure to also add the Amplify-Flutter Authenticator to your `pubspec.yaml` file:
+
+```yaml
+dependencies:
+  amplify_authenticator: ^1.0.0-rc.0
```

Comment:
Need `amplify_flutter` and `amplify_auth_cognito` as well

Reply:
See question below.